api/app/signals/apps/email_integrations/actions/abstract.py

```diff
@@ -0,0 +1,72 @@
+# SPDX-License-Identifier: MPL-2.0
+# Copyright (C) 2021 - 2022 Gemeente Amsterdam
+import logging
+from abc import ABC
+
+from django.conf import settings
+from django.core.mail import send_mail
+from django.template import Context, Template, loader
+
+from signals.apps.email_integrations.models import EmailTemplate
+from signals.apps.email_integrations.utils import make_email_context
+from signals.apps.signals.models import Signal
+
+logger = logging.getLogger(__name__)
+
+
+class AbstractAction(ABC):
+    rule = None
+
+    key = None
+    subject = None
+    from_email = settings.DEFAULT_FROM_EMAIL
+
+    note = None
```

Comment:
Consider explaining for the properties `rule`, `key`, `subject`, and `note` what they mean / are used for.